kodilla-stream/src/main/java/com/kodilla/exception/homework/OrderDoesntExistException.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.exception.homework;
+
+public class OrderDoesntExistException extends Exception {
```

Comment:
moze nadpisac tu jakis konstruktor, np. OrderDoesntExistExcption(String orderNumber) { super("Order " + orderNu,mber + " not found.");}